Composer/packages/client/src/CreationFlow/DefineConversation/index.tsx

```diff
@@ -42,6 +42,7 @@ interface DefineConversationProps {
   focusedStorageFolder?: StorageFolder;
   currentPath?: string;
   bots?: Bots[];
+  shouldPresetName: boolean;
```

Comment:
I understand this fix, but we are adding more and more on this `DefineConversation` component to make sure it works both when creating "bot" and "dialog", even when we are now having very different UI for that, right? 

Can you just decouple it, by not sharing such this component at such a high level, or even just remove it? Let's try to make it easier to figure out what happened when "creating a bot", and what happened when "creating a dialog"